AppServer/google/appengine/api/logservice/logservice_stub.py

```diff
@@ -309,8 +308,8 @@ def _Dynamic_Read(self, request, response, request_id):
         _fill_request_log(requestLog, log, request.include_app_logs())
         result_count += 1
 
-      if result_count == count:
-        response.mutable_offset().set_request_id(requestLog.offset)
+        if result_count == count:
+          response.mutable_offset().set_request_id(requestLog.offset)
```

Comment:
The count is included in the query, so it shouldn't exceed the count. This is just to make sure a cursor is included in the response if we satisfy the count (indicating that there are probably more results available if given the new offset).